setup.py

```diff
@@ -18,12 +18,15 @@ def setup_package():
         install_requires=get_install_requirements("requirements.txt"),
         tests_require=get_install_requirements("test-requirements.txt"),
         packages=find_packages(exclude=["tests*"]),
+        license='MIT',
```

Comment:
```suggestion
        license="MIT",
```
black complains about this.